v1/metrics.yaml

```diff
@@ -1709,6 +1775,24 @@ definitions:
               # format for this is a json array: [{rank: 1, article: <<title>>, views: 123}, ...]
               type: string
 
+  by-country:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type: string
+            access:
+              type : string
+            year:
+              type: string
+            month:
+              type: string
+            countries:
```

Comment:
Should be:

```yaml
countries:
  type: array
  items:
    properties:
      rank:
        type: integer
      # etc ...
```

Reply:
@d00rman If you look at https://github.com/wikimedia/restbase/pull/940/files/0f285f88cae43271a03a41de04d98aad12b740ec#diff-582fe1d61f4f621ead868f3fba21e0aaR1775 you'll see that pageviews-tops definition has the same sort of return definition, and they are metrics with the same return format.